tests/unit/NavigationModelTest.php

```diff
@@ -22,117 +18,117 @@ function setUp() {
 	function testParseLines() {
 		$model = new NavigationModel();
 
-		$cases = array();
+		$cases = [];
 
-		$cases[] = array(
-			'param1' => array("*1", "*1", "*1", "**2", "**2", "****4"),
+		$cases[] = [
+			'param1' => [ "*1", "*1", "*1", "**2", "**2", "****4" ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "param1" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)